stable/prometheus-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 6.18.0
+version: 6.18.1
```

Comment:
According to the semver spec, you should increase the minor version if this is a feature

Reply:
bumped the minor version @vsliouniaev